src/CustomerProfile.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from "react";
+
+export default function CustomerProfile({ customerId }) {
+  const [customerInfo, setCustomerInfo] = useState({});
+  useEffect(() => {
+    console.log("...");
+    fetch(`https://cyf-react.glitch.me/customers/${customerId}`)
+      .then(res => res.json())
+      .then(data => setCustomerInfo(data));
+  }, [customerId]);
```

Comment:
good, you've added the id to the dependency array so that a new request is made whenever it changes, well done :)